runtime/pkg/email/templates/call_to_action.mjml

```diff
@@ -12,8 +12,8 @@
   <mj-body>
     <mj-section>
       <mj-column>
-        <mj-image container-background-color="#FFF" width="62px" src="https://cdn.rilldata.com/email-transactional/rill-logo-white-padding.png" />
-        
+        <mj-image container-background-color="#FFF" width="100px" src="https://cdn.rilldata.com/email-transactional/rill-logo-purple.png" />
```

Comment:
1. The image used here needs to have some white background padding. Otherwise the emails can look pretty weird in some email clients when using dark mode.
2. Any reason to increase size to 100px? If I remember correctly, it can end up looking too large when opening the emails on a phone (which I guess is what most people do).

Reply:
1. Update the image to have a circle padding
2. Size was too small on the browser so increased it 100